packages/shared/src/api/subscribe.ts

```diff
@@ -0,0 +1,12 @@
+import type { ClientTypes as Client } from "../client";
+import { subscribeChannelUnreads, subscribeDMUnreads } from "./unreadsApi";
+
+export const subscribeUnreads = async () => {
+  async function handleUnreadUpdate(unread: Client.Unread) {
+    // db.create("Unread", unread, db.UpdateMode.All);
```

Comment:
Same question here, why is this duplicated? And why comment this out?

Reply:
Yeah this is wonky. Removed the duplicate and added a comment for now, will clean up with the next round of work.